baseapp/baseapp_test.go

```diff
@@ -106,12 +106,12 @@ func aminoTxEncoder() sdk.TxEncoder {
 	return legacytx.StdTxConfig{Cdc: cdc}.TxEncoder()
 }
 
-// simple one store baseapp
+// simple one store baseapp  in progress
 func setupBaseApp(t *testing.T, options ...func(*BaseApp)) *BaseApp {
 	app := newBaseApp(t.Name(), options...)
 	require.Equal(t, t.Name(), app.Name())
 
-	app.MountStores(capKey1, capKey2)
+	app.MountStores(capKey1, capKey2) // test  for panic use key typedifferent from  sdk.StoreKey
```

Comment:
Could you update the comment to something more clear please?

Reply:
Yeah, sure. Just wondering if I should test MountStores for panic using keytype different from sdk.StoreKey?